include/mc_rtc/gui/Form.h

```diff
@@ -189,28 +189,43 @@ struct FormComboInput : public FormElement<FormComboInput, Elements::ComboInput>
   inline FormComboInput(const std::string & name,
                         bool required,
                         const std::vector<std::string> & values,
+                        int def,
                         bool send_index = false)
-  : FormElement<FormComboInput, Elements::ComboInput>(name, required), values_(values), send_index_(send_index)
+  : FormElement<FormComboInput, Elements::ComboInput>(name, required), values_(values), def_(def),
+    send_index_(send_index), has_def_(true)
   {
   }
 
+  inline FormComboInput(const std::string & name,
+                        bool required,
+                        const std::vector<std::string> & values,
+                        bool send_index = false)
```

Comment:
I'm a little worried this creates a potential ambiguity, for example `FormComboInput(name, req, values, true)` might be ambiguous?

It's a little annoying for the user but maybe we need a single `FormComboInput(const std::string &, bool, const std::vector<std::string> &, bool send_index = false, int def = -1)` to avoid this issue